designs/tripleo/osc-tripleo.md

```diff
@@ -135,10 +151,15 @@ TODO: Things to consider:
 2. Persistence in case of container migration
 
 
+
 #### OSC TLS & Secret Data Configuration  
-1. How are TLS certificates provided  
+1. The TLS certificates files are stored in a specific directory and same is indicated over the param section of template.
+Also this path is exposed as an environoment variable, so that this information is available outside of the heat template definition.
+>> Questions: 
+a. It is not clear will $tls_truststore_file info. is passed to Docker run? if not, what additional configurations need to be added to make it pass to Docker run?
+b. In the above declaration we are assuming that $tls_truststore_file info is defined as part of class definition and available in the manifest file. Is it possible to define the logic in manifest file, to copy the certificates to another location? otherwise can we add this logic in the resource definition section in heat template to have this copy logic? what is preferred?
 2. Other secret data and passwords  
-3. Relevant material: https://github.com/openstack/tripleo-heat-templates/blob/master/environments/ssl/enable-tls.yaml  
+3. Relevant material: https://github.com/openstack/tripleo-heat-templates/blob/master/environments/ssl/enable-tls.yaml
```

Comment:
please add as a link?

Reply:
ok, i will check that...

Reply:
I see it is already handled as a link, for example https://github.com/openstack/tripleo-heat-templates/blob/master/environments/ssl/enable-tls.yaml. Did i misinterpret the remark?